_data/people.yml

```
@@ -239,6 +239,19 @@
       icon: github-alt
       name: GitHub
 
+- name: Anna Balla
+  pic: /img/people/ballaa.png
+  dates: 2020-present
```

```suggestion
  dates: 2019-present
```
(since you worked with us in 2019 too!)